web/bfex/components/scraper/profile_scraper.py

```diff
@@ -3,8 +3,16 @@
 from bfex.components.scraper.scrapp import *
 
 class ProfileScraper(Scraper):
+    """ This is what will scrape the text off of the Faculty of Science
+        Find an Expert profile pages for each professor
+    """
 
     def get_scrapps(self):
+        """ Get the data from the website and put it into a scrapp
+            Currently it will gather the scrapps, get the orcid links and
+            the researchid links
```

Comment:
Grammar.

Reply:
Gets the data from the website and put it into a scrapp.

            Currently it will saves the scrapps, get the orcid links and
            the researchid links. 






better?

Reply:
That works yep.

Reply:
@morlick